### PR TITLE
non-author travel award info

### PR DESCRIPTION
**Pull request by giovana-morais**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -125,6 +125,13 @@ conference:
         Registration for the workshop is now open.
         Please visit the <b><a href="/registration">Registration</a></b> page for more
         information on how to register.
+    - title: Non-Author Travel Award Applications!
+      color: success
+      main_only: true
+      text: |
+        We're now accepting non-author travel awards applications.
```
Comment (richa-namballa) [inline _config.yml]: I would change this to "We are now accepting travel award applications for local non-author students"

File: travel_awards/index.md [hunk 2]
```diff
@@ -6,23 +6,35 @@ title: "Travel Awards"
...
 Applicants must meet the following criteria to be considered for an award:
-* They must be an author of a paper that was accepted to LAMIR 2024.
+* **Authors Awards**: They must be an author of a paper that was accepted to LAMIR 2024. [This call is **_CLOSED_**]
+* **Partial Awards (Non-Authors)**: We are extending a limited number of partial travel awards to non-author students who wish to attend and engage in the workshop. [This call is **_OPEN_**]
 * They must be enrolled in a degree-granting university or college program in the 2024-25 academic year.
 
-Applicants will be asked to provide verification of enrollment. This confirmation can be submitted in the form of an enrollment certification document, official/unofficial transcript, a signed letter by a faculty advisor, etc. If you are unsure whether your form of documentation is acceptable, please feel free to email us at <a href="mailto:lamir.workshop@gmail.com">lamir.workshop@gmail.com</a>.
+Applicants will be asked to provide verification of enrollment. This
+confirmation can be submitted in the form of an enrollment certification
+document, official/unofficial transcript, a signed letter by a faculty advisor.
```
Comment (richa-namballa) [inline travel_awards/index.md]: Add the , etc. back

**Review by richa-namballa — APPROVED** [review]
Some minor grammatical suggestions but otherwise looks good.

File: travel_awards/index.md (hunk 2 above)
Comment (giovana-morais) [reply travel_awards/index.md]: ughhhh ok. for the record, i don't like it. hahah

**Pull request merged**